### ISSUE TITLE
Cannot remove docsets containing a space.

### ISSUE THREAD

**Issue by mhoogendoorn** [sunaku/dasht#5]
I installed the .Net Framework documentation:
```bash
$ dasht-docsets
NET Framework
```
They did not include FSharp documentation, so I tried to remove them:
```bash
$ dasht-docsets-remove
Removable docsets (2):

        NET, Framework

echo Remove NET docset [y/N] ?...
echo Remove Framework docset [y/N] ?...
```
The space in the `NET Framework.docset` directory seems to be the culprit.

**Comment by mhoogendoorn** [comment]
If in `dasht-docset-remove` I add:
```bash
IFS="
"
```
right before the `set -- $(dasht-docsets "$@")` it does work since the output of `dasht-docset` only gets split on newlines, not on spaces/tabs. The "Removable docsets" line is still incorrect as the `sed` still replaces spaces by a comma followed by a space.

**Comment by sunaku** [comment]
Thanks for your report! :+1: I'm wondering where that space came from in the first place, because the installer shows that an underscore is used between "NET" and "Framework".  Did the tarball extract out to a folder named with a space instead of underscore?

```sh
$ dasht-docsets-install net
--2016-03-07 10:30:58--  https://kapeli.com/docset_links
Resolving kapeli.com (kapeli.com)... 23.92.22.81
Connecting to kapeli.com (kapeli.com)|23.92.22.81|:443... connected.
HTTP request sent, awaiting response... 304 Not Modified
File ‘/home/sunny/.local/share/dasht/docsets/docset_links’ not modified on server. Omitting download.

Installable docsets (2):

        MarionetteJS, NET_Framework
```

**Comment by sunaku** [comment]
Fixed now in master branch at commit b73f37ce0838ecaec2ee826bca113074d372fcae. :gift: 

I decided to avoid spaces in the first place by introducing a new dasht-docsets-extract(1) script.
